AutoRest/Modelers/Swagger.Tests/Swagger/swagger-data-types.json

```diff
@@ -368,7 +368,7 @@
           "type": "string"
         },
         "color": {
-          "x-ms-enum": "Colors",
+          "x-ms-enum": { "name": "Colors", "expandable": false },
```

Comment:
The name "expandable" isn't really self-explanatory. Maybe something more like "modelAsString"?